modules/fox-spring/src/main/java/com/ensolvers/fox/spring/lightweightcontainer/GlobalApplicationContext.java

```diff
@@ -0,0 +1,122 @@
+ *      Has two main features: look up and build.   <p>
+ *      <p>
+ *      <p>
+ *      * Look up allows to get from Spring context any registered bean  <p>
+ *      Example:  <code> MyComponent component = (MyComponent)GlobalApplicationContext.getBean( "MyComponent" );</code> <p>
+ *      <p>
+ *      <p>
+ *      * Construction allows building on the fly new context by offering legacy configuration xml files. <p>
+ *      Example:  <code> MyComponent component = (MyComponent)GlobalApplicationContext.getBean( "MyComponent" );</code> <p>
+ *
+ *      <p>
+ *
+ */
+@Component
+public class GlobalApplicationContext implements ApplicationContextAware {
+    private static ApplicationContext context;
+
+
+    public static void setGlobalContext( ApplicationContext globalContext ) {
+        Logger.info(GlobalApplicationContext.class, "setGlobalContext [" + globalContext + "]" );
+        context = globalContext;
+    }
+
+    public static Object getBean( String name ) {
+        if ( context == null ) {
+            Logger.error( GlobalApplicationContext.class, "No Spring Context available" );
+            throw new ApplicationContextException( "No Spring Context available" );
+        }
+        return getInstance().getBean(name);
+    }
+
+    public static ApplicationContext getInstance() {
+        return context;
+    }
+
+    public static Collection<String> getContextsFromProperty( InputStream stream ) throws IOException {
+        Vector contexts = new Vector();
+        if ( stream != null ) {
+            BufferedReader reader = new BufferedReader( new InputStreamReader( stream ) );
+            String line;
+            while ( (line = reader.readLine() ) != null ) {
+                contexts.add( line );
+                Logger.debug( GlobalApplicationContext.class, "context line read [" + line + "]" );
+            }
+        }
+
+        return contexts;
+    }
+
+    public static Collection<String> getContextsFromProperty( String propertyClassPath ) throws IOException {
+        InputStream stream = GlobalApplicationContext.class.getResourceAsStream( propertyClassPath );
+        if ( stream == null ) {
+            Logger.error( GlobalApplicationContext.class, "context property not found at ClassPath with path definided as [" + propertyClassPath + "]" );
+            return new Vector();
+        }
+        return getContextsFromProperty( stream );
+    }
+
+    public static String[] applyDevelopContextFilter(String[] paths) throws ApplicationContextException {
+        Vector<String> filtered = new Vector();
+        try {
+            for (String path : paths) {
+                Logger.info(GlobalApplicationContext.class, "configLocations [" + path + "]");
+                Resource[] resources = new PathMatchingResourcePatternResolver().getResources(path);
+                Logger.debug(GlobalApplicationContext.class, "resources obtenidos segun locations [" + resources.length + "]");
```

Comment:
Need to translate everything to english, in addition to document all these methods in english to